master/security/adopt-zero-trust.md

```diff
@@ -0,0 +1,314 @@
+
+You should assign unique identities to microservices even if you happen to know that they access the same things. Your policy will be more readable if the identities correspond to logical components of the application. You can grant them the same permissions easily, and if in the future they need different permissions it will be easier to handle.
+
+After you decide on the set of identities you require, create the Kubernetes Service Accounts, then modify your application configuration so that each Deployment, ReplicaSet, StatefulSet, etc. uses the correct Service Account.
+
+#### Write initial whitelist policies for each service
+
+The final step to establishing your Zero Trust Network is to write the policies for each service in your network. The Application Layer Policy Tutorial gives an overview of setting up policies that allow traffic based on Service Account identity.
+
+For each service you will:
+1. Determine the full set of other identities that should access it.
+1. Add rules to allow each of those flows.
+
+After a pod is selected by at least one policy, any traffic not explicitly allowed is denied. This implements the Zero Trust Network paradigm of an explicit whitelist of expected flows.
+
+#### Determine the full set of identities that should access each service
+
+There are several approaches to determining the set of identities that should access a service. Work with the developers of the application to generate this list and ensure it is correct. One approach is to create a flow diagram of your entire application. A flow diagram is a kind of graph where each identity is a node, and each expected flow is an edge.
+
+Let’s look at an example application.
+
+![zero-trust-app]({{site.baseurl}}/images/zero-trust-app.png)
+
+In this example, requests from end-users all flow through a service called api, where they can trigger calls to other services in the backend. These in turn can call other services. Each arrow in this diagram represents an expected flow, and if two services do not have a connecting arrow, the are not expected to have any network communication. For example, the only services that call the post service are api and search.
+
+For simple applications, especially if they are maintained by a single team, the developers will probably be able to just write down this flow graph from memory or with a quick look at the application code.
+
+If this is difficult to do from memory, you have several options.
+
+1. Run the application in a test environment with policy enabled.
+  1. Look at service logs to see what connectivity has broken. 
```

Comment:
these sub-lists are not rendering correctly. I think you need 4 spaces of indent

Reply:
Fixed.

Reply:
Fixed.
Indents don't work (that gives roman numerals). The only way I could get this to work is to manually indent and add 4 spaces to previous lines.

Reply:
@bmckercher123 the first sublist is missing the first item in the rendered output

Reply:
Good catch! I had a hard time getting that list to render correctly.